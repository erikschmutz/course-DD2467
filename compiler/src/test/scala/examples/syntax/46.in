type a = (float -> float) * (float * float);;