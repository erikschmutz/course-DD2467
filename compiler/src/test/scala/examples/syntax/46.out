Program(
    List(
        TypeDeclaration(a,
        TypeEquation(
            TypeTuple(
                List(
                    TypeFunction(
                        TypePrimitive(float),
                        TypePrimitive(float)
                    ),
                    TypeTuple(
                        List(
                            TypePrimitive(float),
                            TypePrimitive(float)
                        )
                    )
                )
            )
        ),
        List()
        )
    )
)